examples/twittermap/web/app/views/twittermap/index.scala.html

```diff
@@ -32,18 +32,24 @@
     </button>
   </div>
 
-
   <div class="stats">
     <time-series id="chart"></time-series>
   </div>
 
-  <div id='sidebar' ng-init="click = -1" ng-class="{toggled: click === 1}" ng-controller="SidebarCtrl">
+  <!-- button for hiding and showing the sidebar -->
+  <div class="w3-teal">
+    <button type="button" class="btn btn-default btn-sm" onclick="openRightMenu()">
+      <span id="hamburger" class="glyphicon glyphicon-menu-hamburger" aria-hidden="true"></span>
+    </button>
+  </div>
+
+  <div id='sidebar' class="open" ng-init="click = -1" ng-class="{toggled: click === 1}" ng-controller="SidebarCtrl">
     <div class = "col-xs-2">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeRightMenu()">&times;</a>
       <ul class="nav nav-tabs tabs-left">
         <li role="presentation" ng-click="click = 1; showTab('hashtagTab')"><a href="#hashtag" data-toggle="tab"><i class="fa fa-hashtag fa-2x" aria-hidden="true"></i></a></li>
         <li role="presentation" ng-click="click = 1; showTab('sampletweetTab')"><a href="#tweet" data-toggle="tab"><i class="fa fa-twitter fa-2x" aria-hidden="true"></i></a></li>
         <li role="presentation" ng-click="click = 1; showTab('aboutTab')" class="active"><a href="#about" data-toggle="tab"><i class="fa fa-info-circle fa-2x"></i></a></li>
-        <li role="presentation" ng-click="click = click*-1; showOrHideSidebar(click)"><i ng-class="click ===-1 ? 'fa fa-chevron-left fa-2x': 'fa fa-chevron-right fa-2x'"></i></li>
```

Comment:
We need to call this `showOrHideSidebar()` function when the `hamburger` button is clicked, we can call it inside your `openRightMenu()` function.

Reply:
Yeah. I fixed it!